kindlecomicconverter/comic2ebook.py

```diff
@@ -581,6 +593,69 @@ def buildEPUB(path, chapternames, tomenumber, ischunked, cover: image.Cover, len
     buildOPF(path, options.title, filelist, cover)
 
 
+def buildPDF(path, title, cover=None, output_file=None):
+    """
+    Build a PDF file from processed comic images.
+    Images are combined into a single PDF optimized for e-readers.
+    """
+    import io
+    import gc
+    
+    # prepare image list
+    images_path = os.path.join(path, "OEBPS", "Images")
```

Comment:
flattenTree() function may be useful here since all files have unique ordered names.